core-tests/src/test/java/org/mule/runtime/core/MuleEventTestCase.java

```diff
@@ -39,7 +43,10 @@
 
   @Test
   public void testEventSerialization() throws Exception {
-    Event event = getTestEvent("payload");
+    FlowConstruct flowConstruct = MuleTestUtils.getTestFlow(muleContext);
+    Event event = Event.builder(DefaultEventContext.create(flowConstruct, TEST_CONNECTOR))
```

Comment:
Also, when all that is being created is an event from a payload, why does the test need to have all this boiler plate code?